schedule.php

```diff
@@ -1,28 +1,33 @@
 <?php
+header('Content-Type: application/json');
+header('Access-Control-Allow-Origin: *');
+date_default_timezone_set('America/Toronto');
```

Comment:
PHP was complaining that these weren't at the top of the response, so there ya go.